docs/Using-Fleet/fleetd.md

```diff
@@ -262,17 +205,17 @@ This process may take several minutes, as the notarization process completes on
 
 After successful notarization, the generated "ticket" is automatically stapled to the package.
 
-#### Orbit osquery result and status logs
+#### Fleetd osquery result and status logs
 
 If the `logger_path` configuration is set to `filesystem`, Orbit will store osquery's "result" and
```

Comment:
Should this be `Fleetd`?

Reply:
Thanks for catching that, I updated that section.